SPECS/python-apache-libcloud/python-apache-libcloud.spec

```
@@ -0,0 +1,1070 @@
+#
+# spec file for package python-apache-libcloud
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+%global _description \
+Apache Libcloud is a standard Python library that abstracts away \
+differences among multiple cloud provider APIs.
+%global pypi_name apache-libcloud
+
+Name:           python-apache-libcloud
+Version:        3.5.1
+Release:        2
```

Release should always have the distro tag at the end, so we get that nice `.cmX` in the RPM name

```suggestion
Release:        2%{?dist}
```

---

My bad. Done.